app/javascript/packages/document-capture/components/barcode-attention-warning.tsx

```diff
@@ -1,9 +1,11 @@
 import { Button, StatusPage } from '@18f/identity-components';
 import { t } from '@18f/identity-i18n';
 import { trackEvent } from '@18f/identity-analytics';
+import { useContext } from 'react';
 import { removeUnloadProtection } from '@18f/identity-url';
 import type { PII } from '../services/upload';
 import DocumentCaptureTroubleshootingOptions from './document-capture-troubleshooting-options';
+import FeatureFlagContext from '../context/feature-flag';
```

Comment:
same as above re: which context is needed

Reply:
✅ https://github.com/18F/identity-idp/pull/10156#discussion_r1517894124